hack/Dockerfile.unit

```diff
@@ -1,6 +1,7 @@
 FROM registry.hub.docker.com/library/golang:1.13
 
-WORKDIR /usr/local/kubebuilder
 COPY /hack/tools /usr/local/kubebuilder/
+WORKDIR /usr/local/kubebuilder
 RUN ./install_kustomize.sh && \
     ./install_kubebuilder.sh
+RUN go mod download
```

Comment:
you are not copying the go.mod and go.sum files from the root folder here, so the go mod download is only downloading the modules that are in /hack/tools/go.mod, and not the go.mod ones. So you need to add a copy of the top level go.mod and go.sum and then run go mod download with thoses, outside of the kubebuilder folder to avoid conflicts

Reply:
Done!